### PR TITLE
52029 init layouts

### PR DESCRIPTION
**Pull request by trongxt-0611**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52029)

## WHAT (optional)
- create separate layouts for admin and customer

## HOW
- using AdminLTE for admin site

### admin site
Screenshot from 2022-08-11 13-39-55
### customer site
Screenshot from 2022-08-11 13-44-42

### CODE REVIEW

**Comment by haltn-3346** [comment]
oke ròi á

**Comment by trongxt-0611** [comment]
ready

File: app/assets/stylesheets/admin/admin.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/admin controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/admin/admin.scss]: remove

File: app/assets/stylesheets/admin/dashboard.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/dashboard controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/admin/dashboard.scss]: remove

File: app/assets/stylesheets/customer/home.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the customer/home controller here.
+// They will automatically be included in application.css.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/customer/home.scss]: remove

File: app/helpers/admin/dashboard_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module Admin::DashboardHelper
```
Comment (ngocvt-0484) [inline app/helpers/admin/dashboard_helper.rb]: remove

File: app/helpers/customer/home_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module Customer::HomeHelper
```
Comment (ngocvt-0484) [inline app/helpers/customer/home_helper.rb]: remove

**Comment by trongxt-0611** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**